Taskfile.yml

```diff
@@ -153,6 +153,13 @@ tasks:
       - bin/goimports{{exeExt}} -format-only -local=github.com/FerretDB/FerretDB -w .
       - bin/gofumpt{{exeExt}} -w .
 
+  fmt-yaml:
+    desc: "Format yaml"
+    cmds:
+      - find . -name "*.yml" -exec chmod o+w {} \;
+      - docker compose run --rm -v $(pwd):/work prettier --write --parser=yaml "./**/*.yml"
+      - find . -name "*.yml" -exec chmod o-w {} \;
```

Comment:
Thank you a lot for fixing this so quickly!

My last concern is about those `find` commands. We try to make Taskfile more or less OS-independent, so it can be called on Linux, Mac and Windows. With `find` I'm not sure if it would work on Windows the same way.

I understand your idea to set/unset writing permission, but I tried running it locally without those `find` commands, and still, the formatter was able to modify my files. Are there other cases when such permissions must be set?

Reply:
Thanks for your feedback. I am running docker under ubuntu, and without using find I got the following error.
![image](https://user-images.githubusercontent.com/5955201/233772913-51d7422a-e7ad-41f8-8d82-487394742168.png)

On which env did you test successfully even without using find? on Window?

Reply:
How about write a go program to change the permission?

Reply:
Thanks a lot for the explanation! I don't think we need a separate go program, but we can use some power of `Taskfile`!

On Mac, Windows, and Linux I tried something like

```
      - >
        {{if eq OS "linux"}}find . -name "*.yml" -exec chmod o+w {} \;{{end}}
      - docker compose run --rm -v $(pwd):/work prettier --write --parser=yaml "./**/*.yml"
      - >
        {{if eq OS "linux"}}find . -name "*.yml" -exec chmod o-w {} \;{{end}}
```

and it worked! Only linux needs permissions to be changed (windows and mac work as is). And the `{{if eq OS "linux"}}...{{end}}` worked well on my ubuntu. 

So, I approve your PR, feel free to either include these statements, or alternatively I can include them in a separate PR.